src/core/toga/widgets/canvas.py

```diff
@@ -2,19 +2,21 @@
 
 
 class Canvas(Widget):
-    """"""
+    """Create new canvas
```

Comment:
Am I missing something, or are the only changes to canvas docstring changes?

Reply:
Yes, that's correct, only the small change. I originally had the new Context2D and Matrix classes in the canvas module since they were all drawing related. I refactored them out after looking at some of the other modules. I thought having them in separate modules is more consistent with how the other widgets are implemented.

Reply:
Cool. I'm maybe -0 on changing this file just for docs changes, as it clutters the review and the blame history a bit, but I don't think it's a blocker necessarily.

I agree re: having the context be its own file.

Reply:
FWIW - I'm not so concerned about this. This PR is all about the Canvas widget; if implementing it has resulted in some re-organization or documentation updates, it's all "on topic" to my mind.